frontend/src/store/actions/metrics/getApplicationMetrics.ts

```diff
@@ -38,28 +33,17 @@ export const GetInitialData = (
 				globalTime.maxTime / 1000000,
 			]);
 
+			const start = getMicroSeconds({ time: minTime / 1000000 });
```

Comment:
use dayjs here

Reply:
Not sure how dayjs will give the desired value.